plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -218,7 +218,7 @@ func (c *MaxPDVolumeCountChecker) filterVolumes(volumes []v1.Volume, namespace s
 				return fmt.Errorf("PersistentVolumeClaim had no name")
 			}
 			pvc, err := c.pvcInfo.GetPersistentVolumeClaimInfo(namespace, pvcName)
-			if err != nil {
+			if err != nil || pvc == nil {
```

Comment:
Should not `pvc` be `nil` when `err != nil`?

Reply:
There was `pvc == nil` check few lines below, I just moved it here to keep the same behavior as the original predicate.